azure_arc_k8s_jumpstart/multi_distributions/dapr/dapr_k8s_statestore.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# <--- Change the following environment variables according to your Azure service principal name --->
+export appId='<Your Azure service principal name>'
+export password='<Your Azure service principal password>'
+export tenantId='<Your Azure tenant ID>'
+export resourceGroup='<Azure resource group name>'
+export arcClusterName='<The name of your k8s cluster as it will be shown in Azure Arc>'
+export location='<Azure Region>'
+export daprExtensionName='daprextension'
+export redisSKU='Basic'
+export redisVMSize='c0'
+export redisName=secret-store-$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 10 | head -n 1)
```

Comment:
I would add a "do not change comment" like we do in the CAPI script

Reply:
Added